### DISCUSSION TITLE
Blank page seen after login to Dependency Track UI for new user

### DISCUSSION THREAD

**Discussion by Yo150850** [DependencyTrack/dependency-track#1722] [General]
We have AD users (OpenID connect users) configured to login to Dependency Track UI.
When user login first time, he/she comes across blank page as shown in attached screenshot.
We need to manually assign Team membership (e.g. Portfolio managers) to user to grant access to projects. Is there any way to make this Access management more simpler or is it the standard way to grant access.?

![DTlogin](https://user-images.githubusercontent.com/106586177/174021789-17a6da3b-e6b5-49c1-a8fa-73e6d75e9d46.png)

**Comment by curnsey** [comment]
You can map the groups in AAD to the groups in DT. **Ensure you have these two lines in the compose file.** 
```

  - "ALPINE_OIDC_TEAMS_CLAIM=groups"
  - "ALPINE_OIDC_TEAM_SYNCHRONIZATION=true"

```
Without the SYNCHRONIZATION set to true, the groups won't map. Then you just need to create the group in DT that matches the AAD group name, and map that to a Team in DT, and provide that Team with the required permissions.

Edit: properly style code snippet

**Reply by Yo150850** [reply]
Thanks for the answer !!
